CHANGELOG_UNRELEASED.md

```diff
@@ -0,0 +1,17 @@
+# Changelog (unreleased)
+
+To avoid having old PRs put changes into the wrong section of the CHANGELOG,
+new entries now go to the present file as discussed
+[here](https://github.com/math-comp/math-comp/wiki/Agenda-of-the-April-23rd-2019-meeting-9h30-to-12h30#avoiding-issues-with-changelog).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+### Changed
+
+- `eqVneq` lemma is changed from `{x = y} + {x != y}` to
+  `eq_xor_neq x y (y == x) (x == y)` which allows to use as a view and provide
```

Comment:
One last comment: I do not think this counts as a view, so you could write:
- `eqVneq` lemma is changed from `{x = y} + {x != y}` to `eq_xor_neq x y (y == x) (x == y)`, on which a case analysis performs simultaneous replacement of expressions of the form `x == y` and `y == x` by `true` or `false`, while keeping the ability to use it in the way it was used before.

Reply:
done!